app/routes/context_window.py

```
@@ -0,0 +1,940 @@
+        )
+        return CompactionStatusResponse(
+            success=False,
+            status="error",
+            error=str(e)
+        )
+
+
+@router.get("/tasks/{task_id}/progress")
+async def get_task_progress(
+    task_id: str,
+    user_id: str = Depends(get_current_user)
```

Ruff (0.14.14)

[warning] Unused function argument: `user_id`

(ARG001)